kobo/apps/openrosa/apps/logger/migrations/0042_attachment_user_attachment_xform_and_more.py

```diff
@@ -0,0 +1,80 @@
+# Generated by Django 4.2.15 on 2025-03-06 15:16
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+import kobo.apps.openrosa.apps.logger.models.attachment
+import kobo.apps.openrosa.apps.logger.models.xform
+import kpi.fields.file
+from kpi.deployment_backends.kc_access.storage import KobocatFileSystemStorage
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('logger', '0041_add_root_uuid_field_to_instance'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='attachment',
+            name='user',
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name='attachments',
+                to=settings.AUTH_USER_MODEL,
+            ),
+        ),
+        migrations.AddField(
+            model_name='attachment',
+            name='xform',
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.CASCADE,
+                related_name='attachments',
+                to='logger.xform',
+            ),
+        ),
```

Comment:
This migration should only contain change to the Attachment model. Remove all other changes that Django has detected (because misc differences between your environment and the production).
Please change the name of the file too to make it obvious about the changes we are applying.